src/devices/bus/isa/fdc.cpp

```diff
@@ -139,8 +140,37 @@ MACHINE_CONFIG_START(isa8_fdc_superio_device::device_add_mconfig)
 	MCFG_FLOPPY_DRIVE_ADD("fdc:1", pc_hd_floppies, "35hd", isa8_fdc_device::floppy_formats)
 MACHINE_CONFIG_END
 
+isa8_ec1841_0003_device::isa8_ec1841_0003_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock)
+	: isa8_fdc_device(mconfig, ISA8_EC1841_0003, tag, owner, clock)
+	, bus_mouse(*this, "bus_mouse")
```

Comment:
You're mixing data members with and without `m_` prefix because of the base you derive from.   Could you please add the `m_` prefix for consistency?